client/src/jade/next-distro-fe/form/tabs.jade

```diff
@@ -0,0 +1,101 @@
+// Circles which indicates the steps of the form:
+div(style='text-align:center;margin-top:40px;')
+  span.step
+  span.step
+  span.step
+  span.step
+  span.step
+// One "tab" for each step in the form:
+.tab
+  p
+    | NEXT Distro is an online and mail-based harm reduction program. We provide services to individuals who inject drugs that can not or will not access an in-person syringe exchange.
+  p
+    | As part of our work, NEXT has developed a mail-based supply distribution program. We mail you the supplies you would receive if you walked into a traditional syringe exchange office.
+  p
+    | NEXT Distro is a group of trained harm reductionists and activists. Don't hesitate to reach out if you have questions at 
+    a(href='mailto:info@nextdistro.org') info@nextdistro.org
+  p
+    | NEXT provides supplies to individuals who use drugs yet cannot access resources in their own community. To see if your county has a syringe exchange program, please take a look at 
+    u
+      a(href='https://www.naloxoneforall.org/sep', target='_blank') our directory
+    | . To learn more about NEXT, 
+    u
+      a(href='https://nextdistro.org/history', target='_blank') visit our About page
+    | .
+  p
+    | If you choose to enroll with NEXT Distro, the process will take you approximately 10-15 minutes.
+  .field
+    .control
+      button.button.is-primary#nextBtn(type='button', onclick='nextPrev(1)') Start
+      //
```

Comment:
@drakeltheryuujin So commented-out coding blocks like this are a little messy. If you're not using them at all can you delete these blocks?